lib/tapioca.rb

```diff
@@ -15,6 +22,25 @@ def self.silence_warnings(&blk)
   end
 
   class Error < StandardError; end
+
+  SORBET_PATH = T.let("sorbet", String)
+  SORBET_CONFIG = T.let("#{SORBET_PATH}/config", String)
+  TAPIOCA_PATH = T.let("#{SORBET_PATH}/tapioca", String)
+  TAPIOCA_CONFIG = T.let("#{TAPIOCA_PATH}/config.yml", String)
+
+  DEFAULT_COMMAND = T.let("bin/tapioca", String)
+  DEFAULT_POSTREQUIRE = T.let("#{TAPIOCA_PATH}/require.rb", String)
```

Comment:
Maybe we should rename it:
```suggestion
  DEFAULT_POSTREQUIRE_PATH = T.let("#{TAPIOCA_PATH}/require.rb", String)
```

Or:
```suggestion
  DEFAULT_POSTREQUIRE_FILE = T.let("#{TAPIOCA_PATH}/require.rb", String)
```